src/Materials/effectRenderer.ts

```diff
@@ -214,6 +215,9 @@ export class EffectWrapper {
      */
     public effect: Effect;

```

Comment:
Should this now be always going through drawwrapper and not being assignable ?

Reply:
You're right, I have made accessors for `effect`.